lms/templates/static_templates/400.html

```diff
@@ -0,0 +1,18 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
```

Comment:
Seems that this objects (I mean `HTML` and `Text`) are not used anywhere. Am I wrong ?